galasa-extensions-parent/dev.galasa.auth.couchdb/src/test/java/dev/galasa/auth/couchdb/TestCouchdbAuthStoreValidator.java

```diff
@@ -76,6 +78,48 @@ public void validateRequest(HttpHost host, HttpRequest request) throws RuntimeEx
         }
     }
 
+    class GetTokensDatabaseDesignInteraction extends BaseHttpInteraction {
+        public GetTokensDatabaseDesignInteraction(String expectedUri, Object returnedDocument) {
+            this(expectedUri, returnedDocument, HttpStatus.SC_OK);
+        }
+
+        public GetTokensDatabaseDesignInteraction(String expectedUri, Object returnedDocument, int expectedResponseCode) {
+            super(expectedUri, returnedDocument, expectedResponseCode);
+        }
+
+        @Override
+        public void validateRequest(HttpHost host, HttpRequest request) throws RuntimeException {
+            super.validateRequest(host,request);
+            assertThat(request.getRequestLine().getMethod()).isEqualTo("GET");
+        }
+    }
+
+    class UpdateTokensDatabaseDesignInteraction extends BaseHttpInteraction {
+        public UpdateTokensDatabaseDesignInteraction(String expectedUri, String returnedDocument, int expectedResponseCode) {
+            super(expectedUri, returnedDocument, expectedResponseCode);
+            setResponsePayload(returnedDocument);
+        }
+
+        @Override
+        public void validateRequest(HttpHost host, HttpRequest request) throws RuntimeException {
+            super.validateRequest(host,request);
+            assertThat(request.getRequestLine().getMethod()).isEqualTo("PUT");
+        }
+    }
+
+
+
+//   "_id": "_design/docs",
+//   "_rev": "3-9e69612124f138c029ab40c9c9072deb",
+//   "views": {
+//     "loginId-view": {
+//       "map": "function (doc) {\n  if (doc.owner && doc.owner.loginId) {\n    emit(doc.owner.loginId, doc);\n  }\n}"
+//     }
+//   },
+//   "language": "javascript"
+// }    
```

Comment:
Do we need these comments?

Reply:
This comment has moved to TokensDBNameVIewDesign.java bean. Mike suggested its still relevant